src/layers.js

```diff
@@ -12,11 +12,26 @@ exports.layerInfo = {
         thundraHandlerName: 'thundra.handler.wrapper',
         needHandlerDelegation: true,
     },
-    node: {
-        layerName: 'thundra-lambda-node-layer',
-        defaultLayerVersion: '32',
-        needHandlerDelegation: false,
-        customRuntime: true,
+    node: runtime => {
+        const versionStr = runtime.split('nodejs')[1].split('.')[0]
+        const version = Number(versionStr)
+
+        if (version <= 8) {
+            return {
+                layerName: 'thundra-lambda-node-layer',
+                defaultLayerVersion: '32',
+                needHandlerDelegation: false,
+                customRuntime: true,
+            }
+        } else {
+            return {
```

Comment:
Can we add a flag (config), which is disabled by default, to switch back to old layer based approach. So if there is a problem with the new approach, we can offer a workaround